src/StakeRegistry.sol

```diff
@@ -19,7 +19,8 @@ import "src/StakeRegistryStorage.sol";
  * @author Layr Labs, Inc.
  */
 contract StakeRegistry is StakeRegistryStorage {
-    
+    using BitmapUtils for *;
```

Comment:
hm. why not just uint256

Reply:
doesn't make a difference to the bytecode :shrug: